_includes/nav.html

```diff
@@ -8,15 +8,7 @@
 
     <!-- Logo(s) -->
     <a class="logo-container" href="{{ "/" | relative_url }}">
-      <img class="logo" src="{{ "/assets/img/logo.svg" | relative_url }}" alt="Confluent" />
-      <svg class="sub-domain" xmlns="http://www.w3.org/2000/svg" width="90" height="32" viewBox="0 0 180 29">
-        <g fill="none" fill-rule="evenodd">
-          <rect width="100%" height="100%" fill="transparent" rx="5"></rect>
-          <text fill="currentColor" font-family="Roboto" font-size="17.037" font-weight="bold" letter-spacing=".7">
-            <tspan x="11" y="21">KAFKA TUTORIALS</tspan>
-          </text>
-        </g>
-      </svg>
+      <img class="logo" src="{{ "/assets/img/developer-logo.svg" | relative_url }}" alt="Confluent" />
```

Comment:
Curious if we need to copy the logo over? Can we just link it -- https://developer.confluent.io/images/developer-logo.svg

If we can, we can also:
- preconnect domain - to establish connection to domain earlier
- preload images - since this is the first thing, it'll be good if we can display ASAP

Reply:
@hartantothio Tested `preconnect` & `preload` on slow/fast 3g (disabled cache) and not much difference as of now.